### PR TITLE
Added Messages example using both Form and Presentation

### PR DESCRIPTION
**Pull request by mansbernhardt**
Form adds some extensions to work nicer with Presentation using Swift `canImport(Presentation)`. For this to work properly, it seems that Presentation has to have been built before Form is built. The sample in this PR includes a master detail selection bindTo table kit helper that is behind a canImport condition. If you use any on of these few canImport additions you don't seem to be able to just declare normal pod dependencies. 

```swift
 bag += selection.bindTo(tableKit) // bindTo is behind canImport(Presentation)
```

TODO:
- [x] Has a dependency on Presentation 1.1 that has not been released yet.
- [x] We should investigate if the .podspec and Podfile could be expressed differently.

### CODE REVIEW

**Comment by mansbernhardt** [comment]
@nataliq With the release of Presentation 1.1 the sample now builds. But I think we should spend some more time investigate how to setup the pod specs and pod file correctly and add some docs for the case when you need access to the helpers behind canImport.

File: Examples/Messages/Podfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+platform :ios, '11.0'
+
+target 'Example' do
+  use_frameworks!
+
+  pod 'FormFramework/Presentation', :path => '../..'
+  pod 'FormFramework', :path => '../..'
```
Comment (nataliq) [inline Examples/Messages/Podfile]: I think the second line is not needed. Isn't the first line saying "use the version of Form that has additional dependency on Presentation"?

File: Examples/Messages/Example.xcodeproj/xcuserdata/mansbernhardt.xcuserdatad/xcschemes/xcschememanagement.plist [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (mansbernhardt) [reply Examples/Messages/Example.xcodeproj/xcuserdata/mansbernhardt.xcuserdatad/xcschemes/xcschememanagement.plist]: Yes, git ignore was added after this branch was created.,. will rebase and remove.

[the remark this answers is not in the document]
Comment (nataliq) [reply Examples/Messages/Example.xcodeproj/xcuserdata/mansbernhardt.xcuserdatad/xcschemes/xcschememanagement.plist]: @mansbernhardt  also Travis is not picking up this branch as it was created before we added the integration. Maybe push the changes to a new branch? (not sure if closing and reopening will trigger it).

File: Examples/Messages/Podfile (hunk 1 above)
Comment (mansbernhardt) [reply Examples/Messages/Podfile]: For some reason just using `FormFramework/Presentation` does not seem to install and build Form itself? Must be something with how subspecs work?

**Comment by mansbernhardt** [comment]
@nataliq Rebasing seemed to fixed the travis as well.

**Comment by mansbernhardt** [comment]
@nataliq If you like, it would be great if you could investigate the subspec and Podfile setup. These are mentioned in the todo list

**Comment by mansbernhardt** [comment]
@nataliq I think we should merge this? Could you have another look and approve again?

**Pull request merged**